work_queue/src/work_queue_worker.c

```diff
@@ -2,7 +2,7 @@
 Copyright (C) 2008- The University of Notre Dame
```

Comment:
Please commit code changes that you made. For some reason the whitespace of many lines have been changed. I can help you to do it if you don't know how.

Reply:
Oh, it might have been because I did a right-click + format in netbeans....